<?xml version="1.0"?>
<launch>
    <arg name="dataset"              value="hand"/>
    
    <node pkg="tsl" name="tsl_bag" type="tsl_bag" respawn="false" output="screen">
        <rosparam command="load" file="$(find tsl)/params/hand.yaml" if="$(eval arg('dataset') == 'hand')"/>
    </node>
    <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find tsl)/rviz/tsl_hands.rviz" /> -->

</launch>